### DISCUSSION TITLE
Is these games run locally or online?

### DISCUSSION THREAD

**Discussion by zzz6519003** [webrcade/webrcade#71] [Q&A]
thx

**Comment by nn81** [comment]
It's explained on the README. The main goal is online, however it is possible to run locally when you setup a localhost server at your computer.

**Comment by raz0red** [comment]
So, this question could be taken two ways.

It is all running (locally) in your browser (it is not streamed). Is that your question?

Or, is the question as @nn81 answered? Running webrcade locally? If that is the question, there are a couple of routes, but the easiest would be to use the Docker image.

Thanks,
Chris
